mypy/fastparse.py

```diff
@@ -432,14 +445,13 @@ def visit_Assign(self, n: ast35.Assign) -> AssignmentStmt:
         else:
             new_syntax = False
         if new_syntax and self.pyversion < (3, 6):
-            raise TypeCommentParseError('Variable annotation syntax is only '
-                                        'suppoted in Python 3.6, use type '
-                                        'comment instead', n.lineno, n.col_offset)
+            self.fail('Variable annotation syntax is only suppoted in Python 3.6, '
```

Comment:
typo: suppoted (and you'll have to fix a test in check-newsyntax.test to match the correct spelling :-).